test/groovy/MTABuildTest.groovy

```diff
@@ -128,109 +85,138 @@ public class MTABuildTest extends BasePipelineTest {
 
         assert jscr.shell[0] =~ /sed -ie "s\/\\\$\{timestamp\}\/`date \+%Y%m%d%H%M%S`\/g" ".*\/newDir\/mta.yaml"$/
-
-        assert jscr.shell[1].contains("PATH=./node_modules/.bin:/usr/bin")
-
-        assert jscr.shell[1].contains(' -jar /opt/mta/mta.jar --mtar ')
-
         assert mtarFilePath == "${currentDir}/${newDirName}/com.mycompany.northwind.mtar"
-
-        assert jlr.log.contains("[mtaBuild] MTA JAR \"/opt/mta/mta.jar\" retrieved from environment.")
     }
 
+
     @Test
-    void mtaHomeNotSetTest() {
+    void mtaJarLocationNotSetTest(){
 
         new File("${currentDir}/mta.yaml") << defaultMtaYaml()
 
-        def mtarFilePath = mtaBuildScript.call(script: [commonPipelineEnvironment: cpe], buildTarget: 'NEO')
-
-        assert jscr.shell[0] =~ /sed -ie "s\/\\\$\{timestamp\}\/`date \+%Y%m%d%H%M%S`\/g" ".*\/mta.yaml"$/
+        helper.registerAllowedMethod('setupCommonPipelineEnvironment', [Map], {})
 
-        assert jscr.shell[1].contains("PATH=./node_modules/.bin:/usr/bin")
+        mtaBuildScript.call(buildTarget: 'NEO')
 
-        assert jscr.shell[1].contains(' -jar mta.jar --mtar ')
-
-        assert mtarFilePath == "${currentDir}/com.mycompany.northwind.mtar"
-
-        assert jlr.log.contains( "[mtaBuild] Using MTA JAR from current working directory." )
+        assert jscr.shell[1].contains('-jar mta.jar --mtar')
+        assert jlr.log.contains('[mtaBuild] Using MTA JAR from current working directory.')
     }
 
 
     @Test
-    void mtaHomeAsParameterTest() {
+    void mtaJarLocationFromParametersTest(){
 
         new File("${currentDir}/mta.yaml") << defaultMtaYaml()
 
-        def mtarFilePath = mtaBuildScript.call(mtaJarLocation: '/mylocation/mta', buildTarget: 'NEO')
-
-        assert jscr.shell[0] =~ /sed -ie "s\/\\\$\{timestamp\}\/`date \+%Y%m%d%H%M%S`\/g" ".*\/mta.yaml"$/
-
-        assert jscr.shell[1].contains("PATH=./node_modules/.bin:/usr/bin")
-
-        assert jscr.shell[1].contains(' -jar /mylocation/mta/mta.jar --mtar ')
+        mtaBuildScript.call(buildTarget: 'NEO', mtaJarLocation: '/mylocation/mta')
 
-        assert mtarFilePath == "${currentDir}/com.mycompany.northwind.mtar"
-
-        assert jlr.log.contains("[mtaBuild] MTA JAR \"/mylocation/mta/mta.jar\" retrieved from parameters.".toString())
+        assert jscr.shell[1].contains('-jar /mylocation/mta/mta.jar --mtar')
+        assert jlr.log.contains('[mtaBuild] MTA JAR "/mylocation/mta/mta.jar" retrieved from parameters.')
     }
 
 
     @Test
     public void noMtaPresentTest(){
+
         thrown.expect(FileNotFoundException)
 
-        mtaBuildScript.call(script: [commonPipelineEnvironment: cpe],
-                      buildTarget: 'NEO')
+        mtaBuildScript.call(buildTarget: 'NEO')
     }
 
 
     @Test
     public void badMtaTest(){
+
         thrown.expect(ParserException)
-        thrown.expectMessage('while parsing a block mapping')
+        thrown.expectMessage("while parsing a block mapping")
```

Comment:
What is the reason for that change? Since there are no parameters contained in the string the old notation is fine IMO.